src/components/Common/PopUps/BagPopup/BagPopup.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { BagIcon } from '../../../../assets/homePageImg';
+import './BagPopup.scss';
+
+/**
```

Comment:
The comments provided are not necessary for such a simple component. Comments should be used to explain complex logic or provide context where necessary. In this case, the code is straightforward and self-explanatory, so the comments can be omitted.